packages/ui-components/src/components/dropdown-list-item/test/dropdown-list-item.e2e.ts

```diff
@@ -0,0 +1,41 @@
+import { E2EElement, E2EPage, EventSpy, newE2EPage } from '@stencil/core/testing';
+
+describe('KvBreadcrumbItem (end-to-end)', () => {
+	let page: E2EPage;
+
+	afterAll(async () => {
+		await page.close();
+	});
+
+	describe('when the component renders', () => {
+		let dropdownListItemEl: E2EElement;
+		let labelEl: E2EElement;
+		let clickEventSpy: EventSpy;
+
+		beforeEach(async () => {
+			page = await newE2EPage();
+			await page.setContent(`
+				<kv-dropdown-list-item
+					label='Option 1'
+					value='option1'>
+				</kv-dropdown-list-item>`);
+			dropdownListItemEl = await page.find('kv-dropdown-list-item');
+			labelEl = await page.find('kv-dropdown-list-item >>> .item-label');
+		});
+
+		it('should render label', () => {
+			expect(labelEl.innerText).toContain('Option 1');
+		});
+
+		describe('and the user clicks on the item', () => {
```

Comment:
Should we add here a test here, to check if the `dropdown-list-item` element has the class `selected`?

Reply:
Sure.